tools/bump-version

```diff
@@ -11,12 +11,21 @@ program
   .requiredOption('--owner <owner>', 'owner of the current repository, usually "Integreat"')
   .requiredOption('--repo <repo>', 'the current repository, usually "integreat-webapp" or "integreat-react-native-app"')
   .requiredOption('--branch <branch>', 'the current branch')
+  .option('--development-release', 'whether the release is a development release which is not delivered to production')
 
-const commitAndTag = async (versionName, versionCode, { deliverinoPrivateKey, owner, repo, branch }) => {
+const commitAndTag = async (versionName, versionCodeString, {
```

Comment:
Right now `new-version-code`/`versionCodeString` is optional in the commander.js config. Should we require it here?

Reply:
Hmm... than we would have two different scripts because the webapp does not have a version code. Maybe we should really add one in the webapp for consistency. What do you think?

Reply:
Lets do this in a separate task :)